.mailmap

```diff
@@ -19,3 +22,5 @@ Tim Abbott <tabbott@zulip.com> <tabbott@dropbox.com>
 Tim Abbott <tabbott@zulip.com> <tabbott@humbughq.com>
 Tim Abbott <tabbott@zulip.com> <tabbott@mit.edu>
 Tim Abbott <tabbott@zulip.com> <tabbott@zulipchat.com>
+Vishnu KS <yo@vishnuks.com> <hackerkid@vishnuks.com>
+Vishnu KS <yo@vishnuks.com> <yo@vishnuks.com>
```

Comment:
This identity entry is redundant.

Reply:
I added it to convert `Vishnu Ks <yo@vishnuks.com>` to `Vishnu KS <yo@vishnuks.com>`.

Reply:
Ah okay.